File: app/src/main/java/yoyo/app/android/com/yoyoapp/AboutCityFragment.java
package yoyo.app.android.com.yoyoapp;

import android.os.Bundle;
import android.widget.TextView;
import androidx.core.widget.NestedScrollView;
import androidx.fragment.app.Fragment;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import com.github.florent37.materialviewpager.MaterialViewPagerHelper;


public class AboutCityFragment extends Fragment {

    private NestedScrollView nestedScrollView;
    private View view;
    private TextView aboutTextView, naturalTextView, historicalTextView, manMadeTextView,
            foodTextView, climateTextView, topExperienceTextView;

    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
        view = inflater.inflate(R.layout.fragment_about_city, container, false);

        init();
        nestedScrollView = view.findViewById(R.id.scrollView);
        MaterialViewPagerHelper.registerScrollView(getActivity(), nestedScrollView);
        getBundle();

        return view;
    }

    private void getBundle() {
        Bundle bundle = getArguments();
        aboutTextView.setText(bundle.getString(Utils.KEY_BUNDLE_CITY_ABOUT));
        naturalTextView.setText(bundle.getString(Utils.KEY_BUNDLE_CITY_NATURAL_ATT));
        historicalTextView.setText(bundle.getString(Utils.KEY_BUNDLE_CITY_HISTORICAL_ATT));
        manMadeTextView.setText(bundle.getString(Utils.KEY_BUNDLE_CITY_MAN_MADE_ATT));
        foodTextView.setText(bundle.getString(Utils.KEY_BUNDLE_CITY_FOOD_SOUVENIR));
        climateTextView.setText(bundle.getString(Utils.KEY_BUNDLE_CITY_CLIMATE));
        topExperienceTextView.setText(bundle.getString(Utils.KEY_BUNDLE_CITY_FOOD_TOPEXPERIENCE));
    }

    private void init() {
        aboutTextView = view.findViewById(R.id.tv_city_about);
        naturalTextView = view.findViewById(R.id.tv_city_natural_attractions);
        historicalTextView = view.findViewById(R.id.tv_city_historical_attractions);
        manMadeTextView = view.findViewById(R.id.tv_city_man_made_attractions);
        foodTextView = view.findViewById(R.id.tv_city_food_attractions);
        climateTextView = view.findViewById(R.id.tv_city_climate);
        topExperienceTextView = view.findViewById(R.id.tv_city_top_experience);
    }

}

File: app/src/main/java/yoyo/app/android/com/yoyoapp/AboutCityFragment.kt
package yoyo.app.android.com.yoyoapp

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.fragment.app.Fragment
import com.github.florent37.materialviewpager.MaterialViewPagerHelper
import kotlinx.android.synthetic.main.fragment_about_city.view.*


class AboutCityFragment : Fragment() {

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        val view = inflater.inflate(R.layout.fragment_about_city, container, false)

        MaterialViewPagerHelper.registerScrollView(activity, view.scrollView)
        arguments.also {
            view.tv_city_about.text = it?.getString(Utils.KEY_BUNDLE_CITY_ABOUT)
            view.tv_city_natural_attractions.text = it?.getString(Utils.KEY_BUNDLE_CITY_NATURAL_ATT)
            view.tv_city_historical_attractions.text = it?.getString(Utils.KEY_BUNDLE_CITY_HISTORICAL_ATT)
            view.tv_city_man_made_attractions.text = it?.getString(Utils.KEY_BUNDLE_CITY_MAN_MADE_ATT)
            view.tv_city_food_attractions.text = it?.getString(Utils.KEY_BUNDLE_CITY_FOOD_SOUVENIR)
            view.tv_city_climate.text = it?.getString(Utils.KEY_BUNDLE_CITY_CLIMATE)
            view.tv_city_top_experience.text = it?.getString(Utils.KEY_BUNDLE_CITY_FOOD_TOPEXPERIENCE)
        }

        return view
    }
}
